<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupwiseOutstanding</class>
 <widget class="QWidget" name="GroupwiseOutstanding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Groupwise Outstanding Report</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1,0,0,0,1,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="fromDateLabel">
       <property name="text">
        <string>From Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="fromDateDateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="toDateLabel">
       <property name="text">
        <string>To Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="toDateDateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="groupNameLabel">
       <property name="text">
        <string>Group Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="groupLay"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="extrasLayout"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="totalLabel">
       <property name="text">
        <string>Total:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="totalLineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>tableView</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
